x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/types.ts

```diff
@@ -203,3 +203,14 @@ export interface BuildThreatEnrichmentOptions {
   threatLanguage: ThreatLanguageOrUndefined;
   threatQuery: ThreatQuery;
 }
+
+export interface EventCountOptions {
+  esClient: ElasticsearchClient;
+  exceptionItems: ExceptionListItemSchema[];
+  index: string[];
+  language: ThreatLanguageOrUndefined;
+  query: string;
+  filters: unknown[];
+  tuple?: RuleRangeTuple;
```

Comment:
tuple isn't actually optional, it has to be passed in

Reply:
Fixed